ext/random/random.stub.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/** @generate-class-entries */
+
+namespace
+{
+
+    function lcg_value(): float {}
```

Comment:
Why are these functions being moved? This part of the change looks completely unnecessary to me. These functions can continue to be part of ext/standard.